examples/otel-logs-splunk-raw-hec/README.md

```diff
@@ -0,0 +1,31 @@
+# Splunk Raw HEC example
+
+This example showcases how the collector can expose a HTTP endpoint to receive data over the HEC endpoint as a raw payload.
+
+To learn more about HEC raw capabilities, head on to the [HEC documentation](https://docs.splunk.com/Documentation/Splunk/8.2.6/Data/FormateventsforHTTPEventCollector#Raw_event_parsing).
+
+The example runs as a Docker Compose deployment.
+
+Splunk is configured to receive data from the OpenTelemetry Collector using the HTTP Event collector. To learn more about HEC, visit [our guide](https://dev.splunk.com/enterprise/docs/dataapps/httpeventcollector/).
```

Comment:
Suggested change:  "Splunk is configured to receive data from the Collector using the HEC. For more information about HEC, see Configure the Splunk HTTP Event Collector for use with additional technologies."